dags/openshift_nightlies/scripts/run_benchmark.sh

```diff
@@ -14,7 +11,7 @@ done
 setup(){
     mkdir /home/airflow/workspace
     cd /home/airflow/workspace
-    git clone https://github.com/cloud-bulldozer/e2e-benchmarking
+    git clone -b master https://github.com/cloud-bulldozer/e2e-benchmarking
```

Comment:
Isn't the default branch `master`?

Reply:
If we want to call out the branch, it might make sense to make into a env var we can override. Default it to `master`.

Reply:
yeah i did that because I was gonna work on adding a dependency section in the dag config so we can change it there instead of having to mess with the scripts. I was just gonna do that in a separate PR so it didn't block this

Also I tend to do this pretty frequently and this way I can just replace `cloud-bulldozer`